src/components/FunctionDefinition.js

```diff
@@ -5,28 +5,36 @@ import { graphql } from 'gatsby';
 
 const FunctionDefinition = ({ className, arguments: params, returnValue }) => {
   return (
-    <CodeDef className={className}>
-      <CodeDef.Keyword>function</CodeDef.Keyword>{' '}
-      <CodeDef.Bracket>{params.length > 0 ? '(' : '()'}</CodeDef.Bracket>
-      {params.length > 0 && (
-        <CodeDef.Block>
-          {params.map((param, i) => (
-            <div key={i}>
-              <CodeDef.Identifier>
-                {param.type.startsWith('...') ? `...${param.name}` : param.name}
-                :{' '}
-              </CodeDef.Identifier>
-              <CodeDef.Type>{param.type}</CodeDef.Type>
-              {i !== params.length - 1 ? ', ' : ' '}
-              <CodeDef.Comment text={param.description} />
-            </div>
-          ))}
-        </CodeDef.Block>
-      )}
-      {params.length > 0 && <CodeDef.Bracket>)</CodeDef.Bracket>}
-      <CodeDef.Operator> => </CodeDef.Operator>
-      <CodeDef.Type>{returnValue.type}</CodeDef.Type>
-    </CodeDef>
+    (params.length > 0 || returnValue.length > 0) && (
```

Comment:
Totally optional, but you _could_ do a check for the inverse of these cases and do an early return, rather than wrapping whole component in this if statement:

```js
if (!params.length && !returnValue.length) {
  return;
}

return(
  // JSX goes here
);